lib/purl_listener.rb

```diff
@@ -110,6 +110,8 @@ def listen_for_changes
       listener = Listen.to(path,
                            force_polling: true, # because on NFS mount
                            ignore: [/\.lock$/],
+                           latency: 60,         # wait between polling
+                           wait_for_delay: 5,   # see #157: slow down delta between change notice and action due to NFS
```

Comment:
I wonder if just configuring the `wait_for_delay` is enough to solve our problem?